packages/appstore/src/components/banners/nakala-linked-banner/nakala-linked-banner.scss

```diff
@@ -0,0 +1,34 @@
+.product-linked-banner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.6rem;
+    border: 1px solid var(--general-section-1);
+    border-radius: 8px;
+    cursor: pointer;
+    margin-top: 1.8rem;
+    transition: box-shadow 0.3s ease;
+    width: 373px;
```

Comment:
We normally use rem units for everything, 1rem === 10px. Also, not sure maybe it shouldn't be the strict width for a banner